api/src/Repository/ReportRepository.php

```diff
@@ -230,4 +234,41 @@ public function countAllEntities()
             ->createQuery('SELECT COUNT(r.id) FROM App\Entity\Report\Report r')
             ->getSingleScalarResult();
     }
+
+    /**
+     * @return Report[]
+     */
+    public function getAllSubmittedReportsWithin12Months(string $deputyType)
+    {
+        $oneYearAgo = new DateTime('-1 year');
+
+        switch ($deputyType) {
```

Comment:
Considering these are only a single-line expression maybe it's worth changing the `switch` for a `match` as it's shorter and just looks cleaner as you don't need all the breaks

```
$types = match ($deputyType) {
    'LAY' => Report::getAllLayTypes(),
    'PROF' => Report::getAllProfTypes(),
    'PA' => Report::getAllPaTypes(),
    default => [],
};
```

More on PHP8 match vs switch: https://stitcher.io/blog/php-8-match-or-switch